examples/nginx-with-istio.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-with-istio
+  namespace: test
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx-with-istio
+  template:
+    metadata:
+      labels:
+        app: nginx-with-istio
+      annotations:
+        # this will cause istio to inject its sidecar, that telegraf will then monitor
+        sidecar.istio.io/inject: "true"
```

Comment:
As per https://github.com/influxdata/telegraf-operator/issues/33 we should listen for pod annotation `sidecar.istio.io/status` instead